docs/source/tutorial/schema.md

```diff
@@ -5,8 +5,334 @@ description: Start here for the Apollo fullstack tutorial
 
 <h2 id="apollo-server-setup">Set up Apollo Server</h2>
 
+Apollo Server is a library that helps you build a production-ready graph API over your data. It can connect to any data source, including REST APIs and databases, and it seamlessly integrates with Apollo developer tooling.
+
+We need to install two packages when setting up Apollo Server:
+
+```bash
+npm install apollo-server graphql --save
+```
+
+* [apollo-server](https://npm.im/apollo-server): This is the Apollo Server library.
+* [graphql](https://npm.im/graphql): This package is the JavaScript reference implementation for GraphQL. It's needed for Apollo Server to function as intended.
+
+Create an `src` directory and add an `index.js` file.
+
+Now, import `ApolloServer` and `gql` from the `apollo-server` library.
+
+_src/index.js_
+
+```js
+const { ApolloServer, gql } = require('apollo-server');
+```
+
+In the code above, we imported the `ApolloServer` class and `gql` tag from the `apollo-server` package.
+
+* **ApolloServer**: The `ApolloServer` class instantiates and starts a new GraphQL server.
+* **gql**: The `gql` tag is a JavaScript template literal tag that enables syntax highlighting for our schema.
+
 <h2 id="write-schema">Write your graph's schema</h2>
 
-<h2 id="design-schema">Schema design principles</h2>
+Every GraphQL server runs a schema at its core. A schema defines types and their relationships. The specifications of the types of queries that can be run against a GraphQL server are defined in a schema. Let's design the schema for our app.
+
+GraphQL schemas are at their best when they are designed around the needs of client applications. In fact, this concept is called **Schema First Development**, an approach for building applications with GraphQL that involves the frontend and backend teams agreeing on a schema first, which serves as a contract between the UI and the backend before any API development commences.
+
+In our app, we need to provide the following features:
+
+* Fetch all upcoming rocket launches.
+* Fetch a specific launch.
+* A user should be able to login to be authorized to book and cancel launch trips.
+* A user should be able to book launch trips.
+* A user should be able to cancel launch trips.
+
+We'll use these features to derive the form of queries and mutations our schema needs.
+
+* **First feature:** An example of a query for fetching all upcoming rocket launches would look like:
```

Comment:
I see where you're going with this, but I think including the queries and mutations might confuse people who aren't familiar with GraphQL yet. Let's remove them.